pom.xml

```diff
@@ -67,6 +67,8 @@
     <!-- enable or not TestTracker junit5 extension i.e log message when test method is starting -->
     <jetty.testtracker.log>false</jetty.testtracker.log>
 
+    <surefire.base.argLine>-Dfile.encoding=UTF-8 -Duser.language=en -Duser.region=US -showversion -Xmx1g -Xms1g -XX:+PrintGCDetails</surefire.base.argLine>
```

Comment:
Replace `-XX:+PrintGCDetails` with `-Xlog:gc:stderr:time,level,tags`.
The latter will print something like:
```
[2018-12-04T09:37:16.835+0100][info][gc] GC(0) Pause Young (Normal) (G1 Evacuation Pause) 24M->3M(502M) 13,231ms
```
on standard error, which is what we want because also our logging prints on standard error.